tests/python/bindings_test_replace.py

```diff
@@ -94,10 +94,10 @@ def testRandomSelf(self):
         remaining_data = comb_data[remaining_labels_list]
 
         returned_items = hnsw_index.get_items(remaining_labels_list)
-        self.assertSequenceEqual(remaining_data.tolist(), returned_items)
+        self.assertTrue((remaining_data == returned_items).all())
```

Comment:
Maybe add tests to check the output type?

Reply:
I added type check for return results of get_items to bindings_test_getdata.py